routes/api-routes.js

```diff
@@ -5,7 +5,7 @@ var db = require("../models")
 module.exports = function(app){
   //Get the route for profile information for one person
   app.get("/api/members/:id", function(req, res){
-		db.Members.findOne({
```

Comment:
What was the issue with findOne?

Reply:
I don't follow @mbychkowski. Can you please elaborate?